docs/source/services.md

```diff
@@ -0,0 +1,84 @@
+# JupyterHub services
+
+JupyterHub 0.7 adds the notion of Services.
+A Service is a process that interacts with the Hub REST API.
+Services may perform actions such as shutting down user servers that have been idle for some time,
+or registering additional web servers that should also use the Hub's authentication
+and be served behind the Hub's proxy.
+
+There are two main characteristics of services:
+
+1. Is it **managed** by JupyterHub?
+2. Does it have a web server that should be added to the proxy?
+
+If a `command` is specified for launching the service, it will be started and managed by the Hub.
+If a `url` is specified for where the service runs its own webserver,
+it will be added to the Hub's proxy at `/service/:service-name`.
+
+## Managed services
+
+**Managed** services are services that the Hub starts and is responsible for.
+These can only be local subprocesses of the Hub.
+While there are similarities with notebook Spawners,
+there are no plans to support the same spawning abstractions as notebook
+If you want to run these services in docker or other environments,
```

Comment:
The main use-case we've had at Q for something like services is for one-per-hub containers that do things like run [pgbouncer](https://pgbouncer.github.io/) or maintain data volumes that are shared with user notebook containers.

Am I right in thinking that these uses wouldn't be well-suited for services as they're outlined here?  It sounds like I basically want a managed service that isn't a simple local process (I could shell out to `docker run` I suppose, but then I'd have to keep the container open in interactive mode or the process would exit immediately), which is explicitly noted as being unsupported here.